blog/_posts/2023-09-21-scala-survey-2023-announcement.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: blog-detail
+post-type: blog
+by: Kacper Korban, VirtusLab
+title: Scala Survey is back!
+---
+
+![Scala Survey 2023 Announcement]({{ site.baseurl }}/resources/img/scala-survey-2023-announcement.png)
+
+## The Scala Team wants to hear from you again!
+
+Over the last year we used your feedback from the Scala Survey and to improve the language we all love. This year we want to hear from you again. How has your usage of Scala changed over the last year? Which direction would you like for Scala to take? Take part in the Scala Survey 2023 and shape the future of Scala.
+
+[Scala Survey 2023](https://scala-survey-2023.virtuslab.com/)
+
+The Survey will close on **14.10.2023**.
```

Comment:
```suggestion
The Survey will close on **October 14, 2023**.
```

Reply:
Thanks, I forgot that people in the US switch the dates :sweat_smile:

Reply:
It is our national shame.